codemon/CodemonReg.py

```diff
@@ -0,0 +1,28 @@
+import sys
+import os
+import ctypes
+from clint.textui import colored
+
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+
+def codemonReg():
+    print(ROOT_DIR)
+    path = os.path.join(ROOT_DIR, "codemon")
+    if not os.path.exists(path):
+        os.makedirs(path)
+        FILE_ATTRIBUTE_HIDDEN = 0x02
+        ret = ctypes.windll.kernel32.SetFileAttributesW(path, FILE_ATTRIBUTE_HIDDEN)
+    meta = ""
+    name = input("Name : ")
+    cf_user = input("CF Username : ")
+    cf_pass = input("CF Password : ")
+    git_user = input("GitHub Username : ")
+    git_pass = input("GitHub Password : ")
```

Comment:
Please use proper and full variable names such as `codeforces_username`, `codeforces_password`, `github_username` & `github_password`. Also the input prompt should display proper text. For example, 
```
Codemon User Registration

Please enter the following data. Your sensitive information is collected for making your life easier while using Codemon and will be stored on your local machine. It won't be stored or used by Codemon developers for any purpose. 

Codeforces username (without @) : someusername
Codeforces password: somepassword
Github username: someusername
Github password: somepassword